### DISCUSSION TITLE
There's some way to transfer my custom paths to a newer version of the mod?

### DISCUSSION THREAD

**Discussion by ThaigoLOCAO** [Joe91/fun-bots#111] [Q&A] [answered]
I made some paths to Noshahr Canals in the version 2.1.0 RC-1, in the future when release a new version of the mod, will be possible to transfer these and other paths?

**Comment by Joe91** [comment, marked as answer]
Rn you can only send me your mod.db file and I will merge your paths into the master. But there are some ideas on how to automate this in future...
Just sent the file to me on discord with the information what paths to merge.
Thanks a lot for participating in the path-creation!!

If you only want the paths to be saved, you can create a copy of your mod.db and replace it after an update...
